README.md

```diff
@@ -28,3 +34,45 @@ Wekan is an open-source kanban board (task manager and organizer)
 **Private/Public mode:** In private mode, only authorized YunoHost members can access to the wekan. 
 
 **SSO/LDAP:** SSO and LDAP are not configured.
+
+## Configuration
+
+First registered user will be admin, and next ones normal users. If you want other admins too, you can change their permission to admin at Wekan Admin Panel.
+
+## Documentation
+
+ * Official documentation: https://github.com/wekan/wekan/wiki
+
+## YunoHost specific features
+
+#### Supported architectures
+
+* x86-64b - [![Build Status](https://ci-apps.yunohost.org/ci/logs/wekan%20%28Community%29.svg)](https://ci-apps.yunohost.org/ci/apps/wekan/)
+* ARMv8-A - [![Build Status](https://ci-apps-arm.yunohost.org/ci/logs/wekan%20%28Community%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/wekan/)
+* Jessie x86-64b - [![Build Status](https://ci-stretch.nohost.me/ci/logs/wekan%20%28Community%29.svg)](https://ci-stretch.nohost.me/ci/apps/wekan/)
+
+## Limitations
+
+* Any known limitations.
```

Comment:
I think that in fact refers to the two items in "Status" up a few paragraphs in this README ? Not sure what to do exactly

Reply:
As far as I've understood that part of the readme, it's more to explain if there some particular limitations, for example, having a specific network configuration, etc... But for now, I really don't know, maybe, I can remove that part ?